src/CoreWCF.NetTcp/src/CoreWCF/Configuration/TestConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CoreWCF.Configuration
+{
+   public static class TestConfiguration
+    {
+        public static IWebHostBuilder UseBiroj(this IWebHostBuilder webHostBuilder)
+        {
```

Comment:
I think you probably didn't want to include this :grinning:

Reply:
no I want to ... removed the file lol